packages/vkui/src/components/Spinner/Spinner.module.css

```
@@ -20,14 +20,16 @@
  * PanelHeader
  */
 
```

```suggestion
 * CMP:
 * PanelHeader
 */
```

Чтобы к единому формату привести, раз все равно рядышком влезли. 🙈

---

done